src/org/labkey/test/tests/FolderTest.java

```diff
@@ -372,14 +372,14 @@ public void deleteWorkbooksTest() throws Exception
             SelectRowsCommand sr = new SelectRowsCommand("core", "workbooks");
             sr.setColumns(Arrays.asList("EntityId"));
             SelectRowsResponse srr = sr.execute(cn, getProjectName() + "/WorkbookParent");
-            Object rowId = srr.getRows().get(0).get("RowId");
+            Object entityId = srr.getRows().get(0).get("EntityId");
 
             DeleteRowsCommand drc = new DeleteRowsCommand("core", "workbooks");
             Map<String, Object> row1 = new HashMap<>();
-            row1.put("RowId", rowId);
+            row1.put("EntityId", entityId);
             drc.addRow(row1);
 
-            SaveRowsResponse resp = drc.execute(cn, getContainerId());
+            SaveRowsResponse resp = drc.execute(new Connection(WebTestHelper.getBaseURL(), PasswordUtil.getUsername(), PasswordUtil.getPassword()), getContainerId());
```

Comment:
```suggestion
            SaveRowsResponse resp = drc.execute(cn, getContainerId());
```
Not required, but I would keep this change. Might as well re-use the connection from above, regardless of key.

Reply:
Done (reusing existing connection).